### PR TITLE
README.md; Issue Templates; Contributing guide

### PR DESCRIPTION
**Pull request by kimcoleman**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.MD [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+When I follow those steps, I see...
+
+I was expecting to see...
+
+## Isolating the problem
+
+<!-- MARK COMPLETED ITEMS WITH AN [x] -->
+
+- [ ] This bug happens with only the Sitewide Sales plugin active
```
Comment (dparker1005) [inline .github/ISSUE_TEMPLATE.MD:33]: Not sure if we want to say "with only the Sitewide Sales plugin and relevant module plugins active" if an issue is related to a particular integration

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+4. See error
+
+**Screenshots**
+If applicable, please attach a screenshot to make your issue clearer.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Isolating the problem (mark completed items with an [x]):**
+- [ ] I have deactivated other plugins and confirmed this bug occurs when only the Sitewide Sales plugin is active.
```
Comment (dparker1005) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Not sure if we want to say "with only the Sitewide Sales plugin and relevant module plugins active" if an issue is related to a particular integration

Comment (kimcoleman) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I just made this improvement. Thank you!

File: .github/ISSUE_TEMPLATE.MD (hunk 1 above)
Comment (kimcoleman) [reply .github/ISSUE_TEMPLATE.MD]: I just made this improvement. Thank you!

**Pull request merged**